exampleSite/content/_index/portfolio/index.md

```diff
@@ -2,7 +2,4 @@
 date = "2018-07-09"
 fragment = "portfolio"
 weight = 750
-
-title = "Portfolio Fragment"
-subtitle = "Displaying animals with links and modals"
```

Comment:
Probably best to keep the title and subtitles within the current examples and just add one header example on top.